charts/hivemq-operator/values.yaml

```diff
@@ -46,6 +46,10 @@ operator:
       tolerations: []
   # Set this string to a name of a externally managed ConfigMap containing the templates for the operator if you want to customize them.
   templateConfigMap: ""
+  ## Extra environment variables that will be pass to the operator in the form of
```

Comment:
Can you provide me please a couple of examples, why normal environment variables are not enough?